department-of-veterans-affairs/vets-api
# Add refresh token cookie path to client

## Summary
Add refresh token. cookie path  to client_config.  This path will be added to the refresh_token cookie when cookies are generated. If no cookie config is present default to the sign in service refresh path.


## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/55778

[ZinHub: ticket 55778](https://app.zenhub.com/workspaces/identity-5f5bab705a94c9001ba33734/issues/gh/department-of-veterans-affairs/va.gov-team/55778)

[](url)
## Testing done

Testing Steps:
 Download [sign-in-service-client-sinatra](https://github.com/department-of-veterans-affairs/sign-in-service-client-sinatra/tree/main) from gitlab. 
 
1. In Sinatra root file structure, Open rails console.
2.  Run. config.update(update(refresh_token_path:'/test_url')) 
3. Start server - rails s
4. Start sign-in-service-client-sinatra - bin/server
5. login with test user
6. Checked Respone_header - "set-cookie" to see that path: /test_url was added. 


## Screenshots
![Screenshot 2023-06-27 at 10 06 14 AM (2)](https://github.com/department-of-veterans-affairs/vets-api/assets/8038703/aee4a07f-8b8c-4c22-973c-4a08a484c7f5)


## What areas of the site does it impact?
Sign-In Service
